### ISSUE TITLE
[Feature Request] Version in tag

### ISSUE THREAD

**Issue by jean-christophe-manciot** [ionos-cloud/reprepro#5]
First off, thanks for offering this "multiple-versions" feature to reprepro.
Right now, the tag does not reflect the version available in debian/changelog:
```
# git tag --list
5.1.1-multiple-versions-2017-02-10
5.1.1-multiple-versions-2017-02-27
5.1.1-multiple-versions-2017-03-27
5.1.1-multiple-versions-2017-03-28
5.1.1-multiple-versions-2017-03-29
5.1.1-multiple-versions-2017-04-12
```
Changing the tag format to reflect debian/changelog version would help us automate package building, for instance with a format like ```v5.1.1-2.1```.
The branch name is explicit.

**Comment by jean-christophe-manciot** [comment]
Also, for some strange reason, the current tags are not seen by some git commands:
```
git-reprepro# git describe --abbrev=0 --tags
fatal: No tags can describe '450b5a626b2d044d688a663c5de4db98650a235d'.
Try --always, or create some tags.
git-reprepro# git describe --tags
fatal: No tags can describe '450b5a626b2d044d688a663c5de4db98650a235d'.
Try --always, or create some tags.
```

**Comment by bdrung** [comment]
Currently I just touched the `debian/changelog` once to increment the version once with a NMU version (e.g. 5.1.1-2.1), but then didn't updated it. So all `5.1.1-multiple-versions-*` tags have the same Debian version `5.1.1-2.1`.

Do you want me to update the Debian version from `5.1.1-2.1` to `5.1.1+multi+version+20170412-0.1`?

**Comment by jean-christophe-manciot** [comment]
No, my proposal is the other way around which is the current best practice on most debian git repositories:
- tag your 5.1.1-multiple-versions branch with a new version whenever you feel the code is stable enough; 5.1.1-2.2 would be reasonable for the first tag with this new format. It is done with something like this on your local system:
```
git tag -a 5.1.1-2.2 -m "Release version 5.1.1-2.2"
git push --porcelain --progress --tags origin "refs/heads/5.1.1-multiple-versions:refs/heads/5.1.1-multiple-versions"
```
- sync your debian/changelog with that tag by prepending the file with a new header for each new tag.

**Comment by bdrung** [comment]
To be precise, the changes affect the upstream code and are new "upstream" releases instead of just small Debian revision changes. So `5.1.1+mv2-0.1` would be more precise.

**Comment by SwooshyCueb** [comment]
Is this the new upstream location?

**Comment by bdrung** [comment]
No, https://salsa.debian.org/brlink/reprepro is the upstream location.
